tenable/io/v3/vm/folders.py

```diff
@@ -0,0 +1,93 @@
+'''
+Folders
+=======
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`folders <folders>` API endpoints.
+
+Methods available on ``tio.folders``:
+
+.. rst-class:: hide-signature
+.. autoclass:: FoldersAPI
+    :members:
+'''
+from typing import List
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class FoldersAPI(ExploreBaseEndpoint):
+
+    _path = 'api/v3/folders'
+    _conv_json = True
+
+    def create(self, name: str) -> int:
+        '''
+        Create a folder.
+
+        :devportal:`folders: create <folders-create>`
+
+        Args:
+            name:
```

Comment:
please provide the argument type.

Reply:
arg type already mentioned in function definition. so skipped here